adapters/appnexus.go

```diff
@@ -147,9 +164,8 @@ func NewAppNexusAdapter(config *HTTPAdapterConfig, externalURL string) *AppNexus
 	}
 
 	return &AppNexusAdapter{
-		http: a,
-		// TODO: Get new endpoint from sweeney
-		URI:          "http://ib.adnxs.com/openrtb2?member_id=958",
+		http:         a,
+		URI:          "http://ib.adnxs.com/openrtb2",
```

Comment:
We should probably be sending a secure flag in the request that would flip this to the secure endpoint (if that's supported on `/openrtb2`)